src/Components/BankDebitForm/BankDebitProvider.tsx

```diff
@@ -145,7 +146,7 @@ export const BankDebitProvider: FC<Props> = ({
   }
 
   return (
-    <div data-test={`${paymentMethod}_PaymentSection`}>
+    <div data-test={`paymentSection${upperFirst(camelCase(paymentMethod))}`}>
```

Comment:
Updated the data attribute to use camel case according to the [comment](https://github.com/artsy/force/pull/10618#discussion_r931205491)